commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraftBuilder.java

```diff
@@ -25,11 +25,25 @@
     @Nullable
     private CustomFieldsDraft custom;
 
+    private CategoryDraftBuilder(final CategoryDraft d) {
+        name = d.getName();
+        slug = d.getSlug();
+        description = d.getDescription();
+        parent = d.getParent();
+        orderHint = d.getOrderHint();
+        externalId = d.getExternalId();
+        custom = d.getCustom();
```

Comment:
Just a picky question: sometimes you use the field assignment and others you call the draft builder's methods. Any reason?